### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by actuallyasmartname**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -75,7 +75,7 @@ dependencies {
 	javadocClasspath "net.fabricmc:fabric-loader:${project.fabric_loader_version}"
 	javadocClasspath "org.jetbrains:annotations:${project.jetbrains_annotations_version}"
 	javadocClasspath "com.google.code.findbugs:jsr305:3.0.2" // for some other jsr annotations
-	decompileClasspath "org.benf:cfr:0.150"
+	decompileClasspath "org.benf:cfr:0.151"
```
Comment (YanisBft) [inline build.gradle]: This should use Fabric fork of CFR, and put the version in gradle.properties

**Review by YanisBft — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (actuallyasmartname) [reply build.gradle]: Fixed in https://github.com/FabricMC/yarn/pull/2343/commits/39985abb91ac989c3d6876208d2bc5dd34cc78df

**Review by YanisBft — APPROVED** [review]

**Review by Player3324 — APPROVED** [review]
LGTM, thanks!

**Review by liach — APPROVED** [review]

File: .github/workflows/update-base.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ jobs:
     if: ${{ github.event.label.name == 'update-base' }}
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/github-script@v3
+      - uses: actions/github-script@v4
```
Comment (modmuss50) [inline .github/workflows/update-base.yml:12]: I assume this didnt break anything? One way to find out I guess

**Review by modmuss50 — APPROVED** [review]

File: .github/workflows/update-base.yml (hunk 2 above)
Comment (actuallyasmartname) [reply .github/workflows/update-base.yml:12]: Guess it didn't

**Pull request merged**